pkg/common/common.go

```diff
@@ -29,7 +29,8 @@ const (
 	DiskImageName = "disk.img"
 	// ImporterWritePath provides a constant for the cmd/cdi-importer/importer.go executable
 	ImporterWritePath = ImporterVolumePath + "/" + DiskImageName
-
+	// ImporterWriteBlockPath provides a constant for the path where the PV is mounted.
+	ImporterWriteBlockPath = ImporterVolumePath + "/blockDevice"
```

Comment:
Can we make this something that is not /data/blockDevice? Its confusing me to to have /data in the path as /data is for file system mode PVCs. I would go with just /blockDevice or /dev/block or something of that nature so there is no possible mixture between the two.

Reply:
I think that was Michael's suggestion, but i can change this.

Reply:
Actually I just double checked with the kubevirt guys, and they will NOT use a block device if it is not /dev/something so we need it to be /dev/block or something of that nature, as long as its in /dev

Reply:
The PVC mount point does not have to be the same for KubeVirt and CDI.